availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
That one line is 10 characters shorter. There are hundreds of such lines, not just for print_trailer but other functions. Since most of the functions only test for truthy anyway, and very few functions test for '=== false', we could probably cut several seconds off load times when/if we convert all of them. Even on my DSL line, it takes too long to load as is.
And we could probably save thousands of hours of typing in future. :)